src/components/radios/_radios.scss

```diff
@@ -152,7 +152,7 @@
     padding-left: $conditional-padding-left;
     border-left: $conditional-border-width solid $govuk-border-colour;
 
-    &[aria-hidden="true"] {
+    .js-enabled &.is-hidden {
```

Comment:
Should we use the 'is-hidden' class AND aria-hidden incase CSS fails to load but JS doesnt?